lib/jekyll-import/importers/rss.rb

```diff
@@ -89,6 +102,11 @@ def self.write_rss_item(item, options)
           end
 
           f.puts output
+          if canonical_link
+            f.puts <<~HTML
+              <p>Originally posted on <a href="#{item.link}">#{URI.parse(item.link).host}</a>.</p>
```

Comment:
Remove this – the user can add this if they want it since `canonical_url` is in the front matter. It can be easily added to the `post` layout.

Reply:
Hi @parkr, that's correct, it can be easily do able by making simple change it `post` layout. But I would like to have it here due to following reasons:
* it's configurable using `--canonical_link` option
* I believe multiple people wants to have reference to the original post. To make every one of them to change `post` layout it's little too much according to me.

 In case if it's not preferred to inter connect frontmatter header and this footer, maybe adding a new option will be better.